`conda` has issues in `threading.py` only on `darwin-x86-64`

I have tried a _number_ of things, without successfully chasing this down. log [here](https://github.com/pkgxdev/pantry/actions/runs/7747232283/job/21127393124):
```
test
  ▌ env
PKGX_DIR=/Users/runner/.pkgx
***
LOGNAME=runner
HOME=/Users/runner/work/pantry/pantry/homes/conda.org-23.11.0
PKGX_PANTRY_PATH=/Users/runner/work/pantry/pantry
LANG=en_US.UTF-8
PKGX_HOME=/Users/runner
USER=runner
SHLVL=1
PATH=/Users/runner/.pkgx/pkgx.sh/v1.1.6/bin:/Users/runner/.pkgx/conda.org/v23.11.0/bin:/usr/bin:/bin:/usr/sbin:/sbin
_=/usr/bin/env
  ▌ pantry script start
+ cd /Users/runner/work/pantry/pantry/testbeds/conda.org-23.11.0
+ conda --version
+ grep 23.11.0
conda 23.11.0
+ source /dev/stdin
++ conda shell.bash hook
++ export CONDA_EXE=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda
++ CONDA_EXE=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda
++ export _CE_M=
++ _CE_M=
++ export _CE_CONDA=
++ _CE_CONDA=
++ export CONDA_PYTHON_EXE=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/python
++ CONDA_PYTHON_EXE=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/python
++ '[' -z '' ']'
++ export CONDA_SHLVL=0
++ CONDA_SHLVL=0
++ '[' -n '' ']'
++++ dirname /Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda
+++ dirname /Users/runner/.pkgx/conda.org/v23.11.0/venv/bin
++ PATH=/Users/runner/.pkgx/conda.org/v23.11.0/venv/condabin:/Users/runner/.pkgx/pkgx.sh/v1.1.6/bin:/Users/runner/.pkgx/conda.org/v23.11.0/bin:/usr/bin:/bin:/usr/sbin:/sbin
++ export PATH
++ '[' -z '' ']'
++ PS1=
++ conda activate base
++ local cmd=activate
++ case "$cmd" in
++ __conda_activate activate base
++ '[' -n '' ']'
++ local ask_conda
+++ PS1=
+++ __conda_exe shell.posix activate base
+++ /Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda shell.posix activate base
++ ask_conda='PS1='\''(base) '\''
export PATH='\''/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin:/Users/runner/.pkgx/conda.org/v23.11.0/venv/condabin:/Users/runner/.pkgx/pkgx.sh/v1.1.6/bin:/Users/runner/.pkgx/conda.org/v23.11.0/bin:/usr/bin:/bin:/usr/sbin:/sbin'\''
export CONDA_PREFIX='\''/Users/runner/.pkgx/conda.org/v23.11.0/venv'\''
export CONDA_SHLVL='\''1'\''
export CONDA_DEFAULT_ENV='\''base'\''
export CONDA_PROMPT_MODIFIER='\''(base) '\''
export CONDA_EXE='\''/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda'\''
export _CE_M='\'''\''
export _CE_CONDA='\'''\''
export CONDA_PYTHON_EXE='\''/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/python'\'''
++ eval 'PS1='\''(base) '\''
export PATH='\''/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin:/Users/runner/.pkgx/conda.org/v23.11.0/venv/condabin:/Users/runner/.pkgx/pkgx.sh/v1.1.6/bin:/Users/runner/.pkgx/conda.org/v23.11.0/bin:/usr/bin:/bin:/usr/sbin:/sbin'\''
export CONDA_PREFIX='\''/Users/runner/.pkgx/conda.org/v23.11.0/venv'\''
export CONDA_SHLVL='\''1'\''
export CONDA_DEFAULT_ENV='\''base'\''
export CONDA_PROMPT_MODIFIER='\''(base) '\''
export CONDA_EXE='\''/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda'\''
export _CE_M='\'''\''
export _CE_CONDA='\'''\''
export CONDA_PYTHON_EXE='\''/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/python'\'''
+++ PS1='(base) '
+++ export PATH=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin:/Users/runner/.pkgx/conda.org/v23.11.0/venv/condabin:/Users/runner/.pkgx/pkgx.sh/v1.1.6/bin:/Users/runner/.pkgx/conda.org/v23.11.0/bin:/usr/bin:/bin:/usr/sbin:/sbin
+++ PATH=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin:/Users/runner/.pkgx/conda.org/v23.11.0/venv/condabin:/Users/runner/.pkgx/pkgx.sh/v1.1.6/bin:/Users/runner/.pkgx/conda.org/v23.11.0/bin:/usr/bin:/bin:/usr/sbin:/sbin
+++ export CONDA_PREFIX=/Users/runner/.pkgx/conda.org/v23.11.0/venv
+++ CONDA_PREFIX=/Users/runner/.pkgx/conda.org/v23.11.0/venv
+++ export CONDA_SHLVL=1
+++ CONDA_SHLVL=1
+++ export CONDA_DEFAULT_ENV=base
+++ CONDA_DEFAULT_ENV=base
+++ export 'CONDA_PROMPT_MODIFIER=(base) '
+++ CONDA_PROMPT_MODIFIER='(base) '
+++ export CONDA_EXE=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda
+++ CONDA_EXE=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda
+++ export _CE_M=
+++ _CE_M=
+++ export _CE_CONDA=
+++ _CE_CONDA=
+++ export CONDA_PYTHON_EXE=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/python
+++ CONDA_PYTHON_EXE=/Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/python
++ __conda_hashr
++ '[' -n '' ']'
++ '[' -n '' ']'
++ hash -r
+ conda create --yes --name snowflakes biopython
+ local cmd=create
+ case "$cmd" in
+ __conda_exe create --yes --name snowflakes biopython
+ /Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda create --yes --name snowflakes biopython
WARNING conda.exception_handler:print_unexpected_error_report(168): KeyError('active_prefix_name')
Traceback (most recent call last):
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/exception_handler.py", line 17, in __call__
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/main.py", line 83, in main_subshell
    exit_code = do_call(args, parser)
                ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/conda_argparse.py", line 194, in do_call
    result = getattr(module, func_name)(args, parser)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/notices/core.py", line 124, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/main_create.py", line 125, in execute
    return install(args, parser, "create")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/install.py", line 360, in install
    unlink_link_transaction = solver.solve_for_transaction(
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/core/solve.py", line 138, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(
                               ^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/core/solve.py", line 199, in solve_for_diff
    final_precs = self.solve_final_state(
                  ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda_libmamba_solver/solver.py", line 143, in solve_final_state
    in_state = SolverInputState(
               ^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda_libmamba_solver/state.py", line 190, in __init__
    _supplement_index_with_system(virtual)
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/core/index.py", line 177, in _supplement_index_with_system
    for package in context.plugin_manager.get_virtual_packages():
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/manager.py", line 312, in get_virtual_packages
    return tuple(self.get_hook_results("virtual_packages"))
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/manager.py", line 1[89](https://github.com/pkgxdev/pantry/actions/runs/7747232283/job/21127393124#step:5:94), in get_hook_results
    plugins = [item for items in hook() for item in items]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/manager.py", line 189, in <listcomp>
    plugins = [item for items in hook() for item in items]
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/virtual_packages/cuda.py", line 65, in conda_virtual_packages
    cuda_version = cached_cuda_version()
                   ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/virtual_packages/cuda.py", line 60, in cached_cuda_version
    return cuda_version()
           ^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/virtual_packages/cuda.py", line 35, in cuda_version
    queue = context.SimpleQueue()
            ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/context.py", line 113, in SimpleQueue
    return SimpleQueue(ctx=self.get_context())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/queues.py", line 341, in __init__
    self._rlock = ctx.Lock()
                  ^^^^^^^^^^
  File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/context.py", line 68, in Lock
    return Lock(ctx=self.get_context())
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/synchronize.py", line 169, in __init__
    SemLock.__init__(self, SEMAPHORE, 1, 1, ctx=ctx)
  File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/synchronize.py", line 80, in __init__
    register(self._semlock.name, "semaphore")
  File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/resource_tracker.py", line 174, in register
    self._send('REGISTER', name, rtype)
  File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/resource_tracker.py", line 182, in _send
    self.ensure_running()
  File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/resource_tracker.py", line 100, in ensure_running
    if self._lock._recursion_count() > 1:
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: '_thread.RLock' object has no attribute '_recursion_count'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/exception_handler.py", line 165, in print_unexpected_error_report
    get_main_info_str(error_report["conda_info"])
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/main_info.py", line 392, in get_main_info_str
    return "\n".join(("", *(f"{key:>23} : {value}" for key, value in builder()), ""))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/main_info.py", line 392, in <genexpr>
    return "\n".join(("", *(f"{key:>23} : {value}" for key, value in builder()), ""))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/main_info.py", line 352, in builder
    if info_dict["active_prefix_name"]:
       ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
KeyError: 'active_prefix_name'

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/exception_handler.py", line 17, in __call__
        return func(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/main.py", line 83, in main_subshell
        exit_code = do_call(args, parser)
                    ^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/conda_argparse.py", line 194, in do_call
        result = getattr(module, func_name)(args, parser)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/notices/core.py", line 124, in wrapper
        return func(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/main_create.py", line 125, in execute
        return install(args, parser, "create")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/cli/install.py", line 360, in install
        unlink_link_transaction = solver.solve_for_transaction(
                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/core/solve.py", line 138, in solve_for_transaction
        unlink_precs, link_precs = self.solve_for_diff(
                                   ^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/core/solve.py", line 199, in solve_for_diff
        final_precs = self.solve_final_state(
                      ^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda_libmamba_solver/solver.py", line 143, in solve_final_state
        in_state = SolverInputState(
                   ^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda_libmamba_solver/state.py", line 1[90](https://github.com/pkgxdev/pantry/actions/runs/7747232283/job/21127393124#step:5:95), in __init__
        _supplement_index_with_system(virtual)
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/core/index.py", line 177, in _supplement_index_with_system
        for package in context.plugin_manager.get_virtual_packages():
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/manager.py", line 312, in get_virtual_packages
        return tuple(self.get_hook_results("virtual_packages"))
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/manager.py", line 189, in get_hook_results
        plugins = [item for items in hook() for item in items]
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/manager.py", line 189, in <listcomp>
        plugins = [item for items in hook() for item in items]
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/virtual_packages/cuda.py", line 65, in conda_virtual_packages
        cuda_version = cached_cuda_version()
                       ^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/virtual_packages/cuda.py", line 60, in cached_cuda_version
        return cuda_version()
               ^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/conda.org/v23.11.0/venv/lib/python3.11/site-packages/conda/plugins/virtual_packages/cuda.py", line 35, in cuda_version
        queue = context.SimpleQueue()
                ^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/context.py", line 113, in SimpleQueue
        return SimpleQueue(ctx=self.get_context())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/queues.py", line 341, in __init__
        self._rlock = ctx.Lock()
                      ^^^^^^^^^^
      File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/context.py", line 68, in Lock
        return Lock(ctx=self.get_context())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/synchronize.py", line 169, in __init__
        SemLock.__init__(self, SEMAPHORE, 1, 1, ctx=ctx)
      File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/synchronize.py", line 80, in __init__
        register(self._semlock.name, "semaphore")
      File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/resource_tracker.py", line 174, in register
        self._send('REGISTER', name, rtype)
      File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/resource_tracker.py", line 182, in _send
        self.ensure_running()
      File "/Users/runner/.pkgx/python.org/v3.11.7/lib/python3.11/multiprocessing/resource_tracker.py", line [100](https://github.com/pkgxdev/pantry/actions/runs/7747232283/job/21127393124#step:5:105), in ensure_running
        if self._lock._recursion_count() > 1:
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
    AttributeError: '_thread.RLock' object has no attribute '_recursion_count'

`$ /Users/runner/.pkgx/conda.org/v23.11.0/venv/bin/conda create --yes --name snowflakes biopython`

  environment variables:
conda info could not be constructed.
KeyError('active_prefix_name')

An unexpected error has occurred. Conda has prepared the above report.
If you suspect this error is being caused by a malfunctioning plugin,
consider using the --no-plugins option to turn off plugins.

Example: conda --no-plugins install <package>

Alternatively, you can set the CONDA_NO_PLUGINS environment variable on
the command line to run the command without plugins enabled.

Example: CONDA_NO_PLUGINS=true conda install <package>

Upload successful.
error: Uncaught (in promise) Error: 1 null
if (!rv.success) throw new Error(`${rv.code} ${rv.signal}`)
                       ^
    at file:///Users/runner/work/_actions/pkgxdev/brewkit/v1/bin/cmd/test:[106](https://github.com/pkgxdev/pantry/actions/runs/7747232283/job/21127393124#step:5:111):24
    at eventLoopTick (ext:core/01_core.js:64:7)
Error: Process completed with exit code 1.
```

perhaps someone with deeper python-fu has ideas.

--- a/projects/conda.org/context.py.diff
+++ b/projects/conda.org/context.py.diff
@@ -16,7 +16,7 @@ index 5d8d179..dfa8da4 100644
              cache_dir_name = "pkgs32" if context.force_32bit else "pkgs"
              fixed_dirs = (
 -                self.root_prefix,
-+                # self.root_prefix,
                  join("~", ".conda"),
++                self.root_prefix,
              )
              if on_win:

--- a/projects/conda.org/package.yml
+++ b/projects/conda.org/package.yml
@@ -8,13 +8,17 @@ warnings:
 
 versions:
   url: https://repo.anaconda.com/miniconda/
-  match: /Miniconda3-py311_\d+\.\d+\.\d+-1-MacOSX-arm64.sh/
+  match: /Miniconda3-py311_\d+\.\d+\.\d+-\d+-MacOSX-arm64.sh/
   strip:
     - /Miniconda3-py311_/
-    - /-1-MacOSX-arm64.sh/
+    - /-\d+-MacOSX-arm64.sh/
 
 dependencies:
   pkgx.sh: ^1
+
+runtime:
+  env:
+    CRYPTOGRAPHY_OPENSSL_NO_LEGACY: 1
 
 build:
   dependencies:
@@ -24,7 +28,14 @@ build:
     curl.se: '*'
   script:
     - bkpyvenv stage {{prefix}} {{version}}
-    - curl -LS https://repo.anaconda.com/miniconda/Miniconda3-py311_{{version}}-1-${SUFFIX}.sh -o miniconda.sh
+    # FIXME: we need a way to handle suffix versions
+    - |
+      for v in 9 8 7 6 5 4 3 2 1 0; do
+        if curl -fLS https://repo.anaconda.com/miniconda/Miniconda3-py311_{{version}}-$v-${SUFFIX}.sh -o miniconda.sh; then
+          break
+        fi
+      done
+      test -f miniconda.sh
     - chmod +x miniconda.sh
     - ./miniconda.sh -b -f -s -p {{prefix}}/venv
 

